app/src/pages/playground/Playground.tsx

```diff
@@ -214,7 +214,7 @@ const playgroundInputOutputPanelContentCSS = css`
  * This width accomodates the model config button min-width, as well as chat message accordion
  * header contents such as the chat message mode radio group for AI messages
  */
-const PLAYGROUND_PROMPT_PANEL_MIN_WIDTH = 475;
+const PLAYGROUND_PROMPT_PANEL_MIN_WIDTH = 570;
```

Comment:
Without this bump, the chat template delete button will get cut off on small screen sizes now that the prompt selector takes up some of the header's room